src/service.js

```diff
@@ -184,6 +190,168 @@ class Service {
       .catch(errorHandler);
   }
 
+  _createbulk (data, params) {
+    return this._insertMany(data)
+      .then(({ data, errors }) => {
+        if (!Array.isArray(data)) {
+          return { data, errors };
+        }
+        data = data.map(result => (this.lean && result.toObject) ? result.toObject() : result);
+        return { data, errors };
+      })
+      .then(({ data, errors }) => {
+        // Check the errors exists
+        if (!params[this.bulkErrorsKey]) {
+          params[this.bulkErrorsKey] = [];
+        }
+
+        if (!Array.isArray(data)) {
+          // Add the errors results to params.errors
+          params[this.bulkErrorsKey] = params[this.bulkErrorsKey].concat(errors);
+
+          return data;
+        }
+        data = data.map(result => select(params, this.id)(result));
+        // Add the errors results to params.errors
+        params[this.bulkErrorsKey] = params[this.bulkErrorsKey].concat(errors);
+
+        return data;
+      })
+      .catch(errorHandler);
+  }
+
+  _insertMany (data) {
+    const INSERT_MANY_CONVERT_OPTIONS = {
```

Comment:
What if someone needs to change these? They aren't exposed anywhere.

Reply:
Agreed, should potentially be an option that gets passsed in.

Reply:
We can expose these when the need arises.